src/Common/src/Interop/Unix/libssl/Interop.SafeSslHandle.cs

```diff
@@ -18,6 +18,12 @@ public SafeSslContextHandle(IntPtr method)
                 handle = SSL_CTX_new(method);
             }
 
+
+            /// Use this constructor only with valid ssl_ctx pointer.
```

Comment:
>  returning SafeHandle from SSL_get_SSL_CTX should take care of this

I was not able to return a SafeHandle  cannot return SafeHandle from ```SSL_get_SSL_CTX```   -  it was failing with MissingMethodException for .ctor. It seems we need a default constructor.

I hope the following will not break the semantics/usability of SslCtxHandle

      public SafeSslContextHandle() : base(IntPtr.Zero, false) {}




The reason I made this private so as to differentiate between